.github/workflows/publish-to-npm-as-latest.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish to npm as latest version
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+' # non-prerelease tag
+jobs:
+  publish-latest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2.1.5
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - run: npm ci
```

Comment:
I only just noticed this now, after reviewing quite a few of these pull-request but, I don't think we need the `npm ci` step here, there's no harm having it either though

I think I added this step when I was investigating how to publish typescript `.d.ts` files for our components, which meant adding typescript as a dependency and installing it before running `npm publish

As we don't run any pre-publish scripts for our components, we don't need to run `npm ci` in this workflow or the prerelease workflow